.github/labels.yml

```diff
@@ -138,11 +138,6 @@
   description: "Make this PR count for hacktoberfest"
   color: "ff7518"
 
-# This label can be added to accept PRs as part of Hacktoberfest
-- name: "hacktoberfest-accepted"
```

Comment:
I'd like to generally keep this, rather than adding it each year, and even though we want to allow tracks to opt out from the spamtoberfest, we should keep the tag centralized to make it appear the same on the repositories that participate.

Reply:
@kotp It is an org-wide label, this just removed the repo-specific version of it. See https://github.com/exercism/erlang-analyzer/blob/aeb68a8fd4dcadb76e79e8b6642993705a899a7e/.github/labels.yml#L137 for proof that it will still be in the labels.yml file. And see https://github.com/exercism/org-wide-files/blob/main/global-files%2F.github%2Flabels.yml#L137 for the global source.